src/main/cpp/src/get_json_object.cu

```diff
@@ -887,7 +883,7 @@ __device__ thrust::pair<bool, size_t> get_json_object_single(
  * @param options Options controlling behavior
  */
 template <int block_size>
-__launch_bounds__(block_size) CUDF_KERNEL
+__launch_bounds__(block_size, 1) CUDF_KERNEL
```

Comment:
A comment on the choice of 1 here for the launch bounds would be good.  This is a pretty weird number in to see in most cases.